default.nix

```
@@ -40,12 +41,30 @@ let staticHaskellPackages = nixpkgs.pkgsStatic.haskellPackages.override {
   overrides = haskellOverrides;
 }; in
 
+let python = nixpkgs.python311.withPackages(ps: with ps; []); in
+
+let wabt-definitions = nixpkgs.copyPathToStore wabt-tests/.; in
+
+let wabt-tests = nixpkgs.runCommandNoCC "wabt-tests" {
+  buildInputs = [ nixpkgs.wabt python wabt-definitions ];
+  allowedRequisites = [];
+} ''
+  mkdir -p $out
+  python3 ${wabt-definitions}/gen.py
+  cp *.wasm $out
+  for f in $(ls *.wat)
+  do
+    wat2wasm $f -o "$out/$f.wasm"
+  done
+  ''; in
+
```

Why not just inline `${./wabt-tests}` like: 
```suggestion
let wabt-tests = nixpkgs.runCommandNoCC "wabt-tests" {
  buildInputs = [ nixpkgs.wabt python ];
  allowedRequisites = [];
} ''
  mkdir -p $out
  python3 ${./wabt-tests}/gen.py
  cp *.wasm $out
  for f in $(ls *.wat)
  do
    wat2wasm $f -o "$out/$f.wasm"
  done
  ''; in

```